bootstrap/api/v1alpha1/rke2config_types.go

```diff
@@ -171,6 +171,11 @@ type AdditionalUserData struct {
 	// +optional
 	Config string `json:"config,omitempty"`
```

Comment:
Do we also want to add a deprecation notice to **Config**?

Reply:
Wasn’t able to find anything specific for this, just added a comment, is this enough?

Reply:
a comment seems good to me